pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -55,7 +57,13 @@ import (
 	"github.com/openshift/origin/pkg/version"
 )
 
-var LegacyAPIGroupPrefixes = sets.NewString(genericapiserver.DefaultLegacyAPIPrefix, api.Prefix, api.LegacyPrefix)
+var (
+	LegacyAPIGroupPrefixes = sets.NewString(genericapiserver.DefaultLegacyAPIPrefix, api.Prefix, api.LegacyPrefix)
+
+	// request paths that match this regular expression will be treated as long running
+	// and not subjected to the default server timeout.
+	originLongRunningEndpointsRE = "(/|^)buildconfigs/.*?/instantiatebinary$"
```

Comment:
no, but it should be a constant, not a variable, if we're going to follow the pattern kube used.